core/domain/domain.core.yml

```diff
@@ -38,6 +39,9 @@ forms:
   - question_answering_form
 
 slots:
+  metadata:
+    type: unfeaturized
```

Comment:
We don't need a metadata slot (yet?)

Reply:
keeping the slot as we discussed offline